providers/src/airflow/providers/standard/provider.yaml

```diff
@@ -28,8 +28,9 @@ versions:
   - 1.0.0
 
 dependencies:
-  - apache-airflow>=2.10.0
+  - apache-airflow>=2.8.0
   - apache-airflow-providers-common-sql>=1.18.0
+  - virtualenv>=20.26.0
```

Comment:
Mhm, I am thinking about this as dependency... so in new Airflow Standard provider will be a standard install dependency, this has a standard dependency to virtualenv... 
So far ths was always optional, even the examples excluded parts if virtualenv was not available as dependency. Do we really want to make this a pre-requisite?

Personally I think it is not harming but we change the default dependencies here. (Compared to 2.x line where no dependecy was modelled)

Reply:
Yeah. Thought about it too. No harm I think. It's been always installed anyway in most cases. And the problem is that if we add it as provider's optional dependency - then almost never will actually install it, while PythonVirtualEnv operator is one that is pretty used.

So I think we should eventually remove the `virtualenv` extra from airflow and have it always installed.

Another option is - we migh change PythonVirtualenv to use the built-in venv. 

Which actually might be a very good idea. WDYT?

Reply:
COmpared to the authors of virtualenv I don't see a real benefir over the included venv. Might be a clever small change.

Otherwise - much cooler would be if UV is supported. We had some discussions in slack... not that pre-commit gets uv support on main-line bewfore we have Airflow 3 out :-D

Reply:
I'd be for both - venv (only) and uv (optional) support in the new PythonVirtualenv in provider.

Reply:
First part fixed in https://github.com/apache/airflow/pull/43568